Product/Production/Common/CONNECTCoreLib/src/test/resources/hibernate/docrepo.hibernate.cfg.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE hibernate-configuration PUBLIC
+"-//Hibernate/Hibernate Configuration DTD//EN"
+"http://hibernate.sourceforge.net/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+    <session-factory>
+        <property name="show_sql">true</property>
+        <property name="dialect">org.hibernate.dialect.MySQLDialect</property>
```

Comment:
All the properties in hibernate configuration file now start with hibernate. So show_sql will be hibernate.show_sql